src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -135,7 +141,9 @@ public void visitJavadocToken(DetailNode ast) {
         String firstSentence = getFirstSentence(ast);
         final int endOfSentence = firstSentence.lastIndexOf(period);
         if (endOfSentence == -1) {
-            if (!firstSentence.trim().startsWith("{@inheritDoc}")) {
+            if (!firstSentence.trim().startsWith(INHERIT_DOC_LITERAL)
+                && !FIRST_CHARACTER_ASTERIC_PATTERN.matcher(firstSentence.trim())
+                .replaceFirst("").trim().startsWith(INHERIT_DOC_LITERAL)) {
```

Comment:
Should we scan the node tree instead of parsing the pure text?

Regardless, xdoc description should be updated to say we don't require a full summary for this instance.